src/components/CodeBlock.vue

```diff
@@ -54,6 +116,27 @@ pre.code-example[class*='language-'] {
     white-space: pre-wrap; // override prism.js tomorrow-theme default
   }
 
+  &--line-numbered {
+    @apply pl-0;
+  }
+
+  &--fixed {
+    min-height: 20rem;
+    max-height: 35rem;
```

Comment:
Where do these numbers come from? Are they related to the dashboard's content container width? If we can't avoid having these here, it'd be good to comment what they reference/why they're here.

Reply:
These are just the numbers Blaze and I came up with. I will add a comment.